dnsapi/dns_dp.sh

```diff
@@ -199,7 +199,7 @@ _get_root() {
 
 #Usage: method  URI  data
 _rest() {
-  m=$1
+  export m=$1		# e.g. GET|POST
```

Comment:
ok, i see:

if [ "$data" ]; then ...

maybe we should do "if [ "$m" = 'POST' ]; then ...
otherwise the var is really not used. you are right, this var should not be exportet